src/kubelet/group_test.go

```diff
@@ -95,7 +95,28 @@ func TestGroup(t *testing.T) {
 				v1.ResourceEphemeralStorage: *resource.NewQuantity(18211586048, resource.BinarySI),
 				v1.ResourceMemory:           *resource.NewQuantity(2033283072, resource.BinarySI),
 			},
-			Conditions:    nil,
+			Conditions: []v1.NodeCondition{
```

Comment:
Again, could be added together with original code, which adds `Conditions: nil`, creating extra context for the reviewer.